File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/appsync/tags/BoomOrBustTag.java
package com.devingotaswitch.rankings.domain.appsync.tags;

import com.devingotaswitch.utils.Constants;

import java.util.Arrays;
import java.util.HashSet;

public class BoomOrBustTag extends Tag {
    public BoomOrBustTag(int count) {
        super(count, Constants.BOOM_OR_BUST_TAG, Constants.BOOM_OR_BUST_TAG_REMOTE, new HashSet<>(Arrays.asList(Constants.QB, Constants.RB, Constants.WR, Constants.TE)));
    }
}

File: FFRv2/app/src/main/java/com/devingotaswitch/rankings/domain/appsync/tags/BoomOrBustTag.kt
package com.devingotaswitch.rankings.domain.appsync.tags

import com.devingotaswitch.utils.Constants
import java.util.*

class BoomOrBustTag(count: Int) : Tag(count, Constants.BOOM_OR_BUST_TAG, Constants.BOOM_OR_BUST_TAG_REMOTE, HashSet(Arrays.asList(Constants.QB, Constants.RB, Constants.WR, Constants.TE)))